secure/json-rpc/src/lib.rs

```diff
@@ -148,6 +148,34 @@ impl JsonRpcClient {
         }
     }
 
+    /// Retrieves the status of a transaction on a given account.
+    pub fn get_transaction_status(
+        &self,
+        account: AccountAddress,
+        sequence_number: u64,
+    ) -> Result<Option<TransactionView>, Error> {
+        let method = "get_account_transaction".into();
+        let params = vec![
+            Value::String(account.to_string()),
+            json!(sequence_number),
+            json!(false),
+        ];
+        let response = self.execute_request(method, params);
+
+        match response.status() {
+            200 => {
+                let response = response.into_string()?;
```

Comment:
Nit: Could probably move `response.into_string()?` before the match.

Reply:
you could if into_string didn't consume response leaving no status to be read :/